forge-gui/res/cardsfolder/upcoming/phyrexian_harvester.txt

```diff
@@ -0,0 +1,13 @@
+Name:Phyrexian Harvester
+ManaCost:R R R R
+Types:Creature Phyrexian Horror
+PT:5/5
+K:Menace
+T:Mode$ DamageDone | ValidTarget$ Card.Self | Execute$ TrigSeek | TriggerDescription$ Whenever CARDNAME is dealt damage, seek that many nonland cards. At the beginning of your next end step, discard those cards.
+SVar:TrigSeek:DB$ ChangeZone | Origin$ Library | Destination$ Hand | AtRandom$ True | NoShuffle$ True | Mandatory$ True | NoLooking$ True | NoReveal$ True | ChangeNum$ X | ChangeType$ Card.nonLand | RememberChanged$ True | SubAbility$ DBDelay
+SVar:DBDelay:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigDiscardExiled | RememberObjects$ Remembered | TriggerDescription$ At the beginning of the next end step, discard those cards.
```

Comment:
The cleanup should happen as part of the normal trigger already

Reply:
You still need to do it manually though

Reply:
You mean between line 8 and 9 ?

Reply:
Yea that's when you don't need it anymore

Reply:
nooo, don't overcomplicate it like that haha

just do it as normal sub after you created the delayed trigger

Reply:
oops forgot you get set cleanup on subability

Reply:
Remove `SubAbility$ DBDiscardExiled`